src/main/resources/views/CommercialVehicles/MSVA30.hbs

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+<head>
+    <title>IVA30</title>
+    <style type="text/css" media="print, screen">
+        {{~> assets/stylesheets/iva-msva-30 this}}
+    </style>
+</head>
+
+<body>
+
+<div class="first-page-header-left">
+    <div class="header-serial-no">
+        <div class="header-serial-no-title">
+            <span id="serial-no"> <b>Serial Number: </b> <!-- Insert data here --> </span>
```

Comment:
why is there a comment in the span? Same for the table below

Reply:
for when the data is wired up, they know where to put it. comment will be removed by actual data.